src/handlers/initiateDataRequest.spec.ts

```diff
@@ -80,10 +100,25 @@ describe('initate data request handler', () => {
     expect(initiateDataTransfer).toHaveBeenCalledWith(testDataRequest)
   })
 
-  it('returns 400 response when request is invalid', async () => {
+  it('returns 400 response when request signature is invalid', async () => {
+    jest.spyOn(global.console, 'warn')
+    givenSignatureIsInvalid()
+    expect(await callHandlerWithBody()).toEqual({
+      statusCode: 400,
```

Comment:
It might be nice to break the execution out of the assertion (we do it in a few places in this file) and separate the setup, execution and assertion with an empty line.

Reply:
changed the code to adhere to this format in latest commit